src/main/java/com/ldtteam/structurize/util/WorldRenderMacros.java

```diff
@@ -1093,7 +1089,7 @@ private RenderTypes(final String nameIn,
                 .setTextureState(NO_TEXTURE)
                 .setShaderState(POSITION_COLOR_SHADER)
                 .setTransparencyState(TRANSLUCENT_TRANSPARENCY)
-                .setDepthTestState(LEQUAL_DEPTH_TEST)
+                .setDepthTestState(AlwaysDepthTestStateShard.ALWAYS_DEPTH_TEST)
```

Comment:
This might (will) require changes elsewhere, cuz sorting. I kinda don't care whether we render whole or not, but it might be kinda noisy if too many are on screen

Reply:
we'll see if it gets too noise with multiple schematics, though the tag tool rendering does require it to be able to see tagged blocks behind walls

Reply:
The single block highlights need adjusting, Mirality already had some decent ideas in mcol